src/features/conductor/flagConductorEvaluatorUrl.ts

```diff
@@ -1,7 +1,7 @@
-import { createFeatureFlag } from 'src/commons/featureFlags';
+import { createFeatureFlag } from '../../commons/featureFlags';
 
 export const flagConductorEvaluatorUrl = createFeatureFlag(
   'conductor.evaluator.url',
-  'https://fyp.tsammeow.dev/evaluator/worker.js',
+  'https://fyp.tsammeow.dev/evaluator/0.2.0/worker.js',
```

Comment:
In the future, will this be hardcoded still?

Reply:
no, language data will be imported and bundled from `language-directory` repo